src/components/form-elements/draft-js-mention-selector/createMentionTimestampSelectorState.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { ContentState, EditorState, Modifier, SelectionState } from 'draft-js';
+import DraftMentionDecorator from './DraftMentionDecorator';
+import { UNEDITABLE_TIMESTAMP_TEXT } from './utils';
+import { convertMillisecondsToHMMSS } from '../../../utils/timestampUtils';
+
+// returns data for first mention in a string
+const getMentionFromText = (text: string) => {
+    // RegEx.exec() is stateful, so we create a new regex instance each time
+    const mentionRegex = /([@＠﹫])\[(\d+):([^\]]+)]/gi;
+    const matchArray = mentionRegex.exec(text);
+    if (!matchArray) {
+        return null;
+    }
+    const [fullMatch, triggerSymbol, id, name] = matchArray;
+    const start = matchArray.index;
+    const end = start + fullMatch.length;
+    const data = { id, name, content: `${triggerSymbol}${name}` };
+    return { start, end, data };
+};
+
+const getTimestampFromText = (text: string) => {
+    const timestampRegex = /#\[timestamp:(\d+),versionId:(\d+)\]/;
+    const matchArray = text.match(timestampRegex);
+    if (!matchArray) {
+        return null;
+    }
+    const [fullMatch, timestamp, versionId] = matchArray;
+
+    const timestampInMilliseconds = parseInt(timestamp, 10);
+    const timestampToDisplay = convertMillisecondsToHMMSS(timestampInMilliseconds);
+    const start = text.indexOf(fullMatch);
+    const end = start + fullMatch.length;
+    const data = {
+        timestampInMilliseconds: parseInt(timestamp, 10),
```

Comment:
should be able to do `timestampInMilliseconds,` since you have the variable above